src/main/java/org/jabref/gui/fieldeditors/LinkedEntriesEditor.java

```diff
@@ -18,16 +18,19 @@
 
 public class LinkedEntriesEditor extends HBox implements FieldEditorFX {
 
-    @FXML private final LinkedEntriesEditorViewModel viewModel;
-    @FXML private TagBar<ParsedEntryLink> linkedEntriesBar;
+    @FXML
+    private final LinkedEntriesEditorViewModel viewModel;
+    @FXML
+    private TagBar<ParsedEntryLink> linkedEntriesBar;
 
     public LinkedEntriesEditor(Field field, BibDatabaseContext databaseContext, AutoCompleteSuggestionProvider<?> suggestionProvider, FieldCheckers fieldCheckers) {
         this.viewModel = new LinkedEntriesEditorViewModel(field, suggestionProvider, databaseContext, fieldCheckers);
 
         ViewLoader.view(this)
-                  .root(this)
```

Comment:
I think, this indent change is an Eclipse vs. IntelliJ-thing. I won't nitpick here. Next time, please only commit code changes, not just reformat changes.